webui/react/src/pages/TaskList.tsx

```diff
@@ -64,7 +74,12 @@ const TaskList: React.FC = () => {
     <Page title="Tasks">
       <div className={css.base}>
         <div className={css.header}>
-          <div />
+          <Input
+            allowClear
+            className={css.search}
+            placeholder="search id and name"
```

Comment:
blocking: From your screenshot, this placeholder appears to not be that useful (though granted it looks like a non-standard resolution). Maybe either a tooltip or make this shorter, and let's use sentence casing: "ID or name" (actually, "description" ?)